src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -814,6 +814,15 @@ def parse(self, stacktrace: str) -> CrashInfo:
           state,
           new_type='Kernel failure\nGeneral-protection-fault')
 
+      # GPU Failure.
+      self.update_state_on_match(
+        GPU_PROCESS_FAILURE,
```

Comment:
i think you need to run python butler.py format to fix indentation (see ci/cd failure).

Reply:
Done.